packages/instrumentation-neo4j/test/neo4j.spec.ts

```diff
@@ -0,0 +1,419 @@
+import 'mocha';
+import expect from 'expect';
+import { InMemorySpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { NodeTracerProvider } from '@opentelemetry/node';
+import { context, setSpan, SpanStatusCode } from '@opentelemetry/api';
+import { AsyncHooksContextManager } from '@opentelemetry/context-async-hooks';
+import { Neo4jInstrumentation } from '../src';
+import { assertSpan } from './assert';
+import { DatabaseAttribute } from '@opentelemetry/semantic-conventions';
+import { normalizeResponse } from './test-utils';
+import { map, mergeMap } from 'rxjs/operators';
+import { concat } from 'rxjs';
+
+const instrumentation = new Neo4jInstrumentation();
+instrumentation.enable();
+instrumentation.disable();
+
+import neo4j, { Driver } from 'neo4j-driver';
+
+describe('neo4j instrumentation', () => {
+    const provider = new NodeTracerProvider();
+    const memoryExporter = new InMemorySpanExporter();
+    const spanProcessor = new SimpleSpanProcessor(memoryExporter);
+    provider.addSpanProcessor(spanProcessor);
+    instrumentation.setTracerProvider(provider);
+    let driver: Driver;
+
+    const getSpans = () => memoryExporter.getFinishedSpans();
+    const getSingleSpan = () => {
+        const spans = getSpans();
+        expect(spans.length).toBe(1);
+        return spans[0];
+    };
+
+    before(async () => {
+        driver = neo4j.driver('bolt://localhost:7687', neo4j.auth.basic('neo4j', 'flows'), {
```

Comment:
Are we connecting to a real server on localhost?
This can be dangerous. `ioredis` has `test:local` command in package.json so regular call to test is not executing something with side-effect.
What do you think about doing something similar in our repo?
We also need to set it up in CI.

Reply:
i don't have an easy way to do it, so I have to vote no..